README.md

```diff
@@ -165,15 +165,15 @@ git checkout master
 8. Add the next minor release to the ``CHANGES`` file:
 
 ```
-## 1.10.0 (unreleased)
+## 1.11.0 (unreleased)
 ```
 
 9. Update the version in ``Client.php``:
 
 ```php
 class Raven_Client
 {
-    const VERSION = '1.10.x-dev';
+    const VERSION = '1.11.x-dev';
```

Comment:
On the section above this one the referenced version has been updated to `2.x.x` but here it's still `1.x.x`